java-generator/core/src/test/resources/io/fabric8/java/generator/approvals/ApprovalTest.testKeycloakCrd.approved.txt

```diff
@@ -61,6 +70,12 @@ public class KeycloakSpec implements io.fabric8.kubernetes.api.model.KubernetesR
         this.externalAccess = externalAccess;
     }
 
+    /**
+     * Controls external database settings. Using an external database requires providing a secret containing credentials as well as connection details. Here's an example of such secret:
+     *      apiVersion: v1     kind: Secret     metadata:         name: keycloak-db-secret         namespace: keycloak     stringData:         POSTGRES_DATABASE: <Database Name>         POSTGRES_EXTERNAL_ADDRESS: <External Database IP or URL (resolvable by K8s)>         POSTGRES_EXTERNAL_PORT: <External Database Port>         # Strongly recommended to use <'Keycloak CR Name'-postgresql>         POSTGRES_HOST: <Database Service Name>         POSTGRES_PASSWORD: <Database Password>         # Required for AWS Backup functionality         POSTGRES_SUPERUSER: true         POSTGRES_USERNAME: <Database Username>      type: Opaque
```

Comment:
It's a shame that here the format got lost 😞

Reply:
Yep, thanks. I am going to have at least a quick look at this.
Seems like the newline (`\n`) char [1] is processed properly, then there in the YAML description there are actually some backslahe ones which are skipped (I didn't check the internal implementation yet but maybe it's because it is taken as an escape char). WDYT @andreaTP ?

Reply:
Double-checked, the formatting is pretty broken in the CRD itself.
Your code is correctly preserving the original escapes.

Reply:
> Double-checked, the formatting is pretty broken in the CRD itself.

Thanks @andreaTP - just logged an issue there, see https://github.com/keycloak/keycloak/issues/10035